### ISSUE TITLE
Recreate timezonefinder 4.3.0 and 4.3.1 wheels

### ISSUE THREAD

**Issue by Lauszus** [piwheels/packages#109]
Package name: timezonefinder
Link to PyPI page: https://pypi.org/simple/timezonefinder/
Link to piwheels page: https://www.piwheels.org/simple/timezonefinder/
Version: >= 4.3.0
Python version: 3.5
I am the maintainer: No
More information:

Is it possible for you to recreate the timezonefinder 4.3.0 and 4.3.1 wheels, as they were renamed due to the following bug: https://github.com/MrMinimal64/timezonefinder/issues/100#issuecomment-621213664.

Basically the issue is that `timezonefinder-4.3.0-py3-none-any.whl` and `timezonefinder-4.3.1-py3-none-any.whl` are NOT compatible with Python 3.5 and below, so pip will fail when trying to install the wheel with the following error:

```bash
ERROR: Package 'timezonefinder' requires a different Python: 3.5.3 not in '>=3.6'
```

**Comment by bennuttall** [comment]
Can't recreate wheels with different tags, not sure how they did that on PyPI?

Instead I've removed the wheels for those versions from piwheels, so that piwheels users will just pick up wheels from PyPI. Have also marked this package as "skip" so that new releases won't make it to piwheels, just in case, until we have implemented Requires-Python https://github.com/piwheels/piwheels/issues/208

**Comment by Lauszus** [comment]
@bennuttall I guess he removed them and reuploaded new ones?

Anyway thanks for removing them!
